shanghai/client.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, host, port, queue: asyncio.Queue, ssl=False, loop=None):
     def sendline(self, line):
         self.writer.write(line.encode('utf-8') + b'\r\n')
 
+    def sendcmd(self, command, params=None):
```

Comment:
yup, should be *params, imho. But `sendcmd(self, command, *params)`. the command is not optional.